src/models/User.ts

```diff
@@ -0,0 +1,11 @@
+const { Schema, model } = require('mongoose');
```

Comment:
use import

Reply:
It's fixed. I'm not sure if what i've added is enough but it should work.